.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
I don't think we want this file in the repo - it looks like it snuck in a couple of weeks ago.